SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -33,6 +33,12 @@ public class AnalysisConfig
         ///  See https://jira.sonarsource.com/browse/SONARMSBRU-100 for details.</remarks>
         public string SonarRunnerWorkingDirectory { get; set; }
```

Comment:
We discussed this offline: is this comment still "live", and if so why do we need two properties set to the same value?